sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -125,9 +125,9 @@ ClientSecretCredential credential = new ClientSecretCredentialBuilder()
 // {your-namespace}.servicebus.windows.net
 String host = "<< EVENT HUBS HOST >>"
```

Comment:
Now that I see this, should we change it to `String fullyQualifiedDomainName = ....`

Reply:
Fixed.